File: backend/src/main/kotlin/com/logisty/core/domain/model/Fleet.kt
package com.logisty.core.domain.model

import com.logisty.core.domain.model.values.FleetId
import com.logisty.core.domain.model.values.FleetName

data class Fleet(
    val fleetId: FleetId,
    val fleetName: FleetName,
)

File: backend/fleet/src/Domain/Model/Fleet.php
<?php

namespace App\Domain\Model;

use Core\Domain\Value\Fleet\FleetId;
use Core\Domain\Value\Fleet\FleetName;
use DateTimeImmutable;

final readonly class Fleet
{
    public function __construct(
        public FleetId           $fleetId,
        public FleetName         $fleetName,
        public DateTimeImmutable $createdAt,
        public DateTimeImmutable $updatedAt,
    ) {}
}
